h5p_proplist_test.go

```diff
@@ -102,7 +102,78 @@ func TestDeflate(t *testing.T) {
 	}
 }
 
-func save(fn, dsn string, dims []uint, dclp *PropList) ([]float64, error) {
+func TestChunkCache(t *testing.T) {
+	DisplayErrors(true)
+	defer DisplayErrors(false)
+	var (
+		fn    = "test_chunk_cache.h5"
+		dsn   = "dset_chunk_cache"
+		dims  = []uint{1000, 1000}
+		cdims = []uint{100, 100}
+	)
+	defer os.Remove(fn)
+
+	dcpl, err := NewPropList(P_DATASET_CREATE)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer dcpl.Close()
+	err = dcpl.SetChunk(cdims)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	cdims_, err := dcpl.GetChunk(len(cdims))
+	if err != nil {
+		t.Fatal(err)
+	}
+	for i, cdim := range cdims_ {
+		if cdim != cdims[i] {
+			t.Fatalf("chunked dimensions mismatch: %d != %d", cdims[i], cdim)
+		}
+	}
+
+	data0, err := save(fn, dsn, dims, dcpl)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	dapl, err := NewPropList(P_DATASET_ACCESS)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer dapl.Close()
+
+	nslots, nbytes, w0, err := dapl.GetChunkCache()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	nslots_, nbytes_, w0_ := nslots*4, nbytes*2, w0/3
```

Comment:
not too fond of these `xyz_` variables.
could we find better names?

Reply:
Fixed